qa/tasks/cephadm.py

```diff
@@ -1133,3 +1135,51 @@ def task(ctx, config):
 
         finally:
             log.info('Teardown begin')
+
+
+def add_mirror(s, mirror: str):
```

Comment:
Also, add doc to the method with parameter description

Reply:
Hm.. the code looks like do not add any mirror it just generations a another string based on `s`. And it actually not adding (appending, prepending) it is just set, so maybe:
`_override_containers_registry_mirror` or so.